data_import/serializers.py

```diff
@@ -5,6 +5,20 @@
 from .models import DataFile
 
 
+def serialize_datafile_to_dict(datafile):
+    """
+    Serialize the datafile for storage in logs.
+    """
+    if not datafile:
+        return dict()
+
+    datafile_serializer = DataFileSerializer(datafile).data
```

Comment:
Minor issue, but there's bad clarity name choice here: this isn't a serializer, it's a serialized datafile.